docs/source/conf.py

```diff
@@ -122,6 +115,16 @@
 html_logo = 'mesa-logo-200.png'
 html_favicon = 'mesa-favicon.png'
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+html_context = {}
+
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
+    html_context["READTHEDOCS"] = True
```

Comment:
I'm very late to this but a slightly cheeky version of this is
```python
html_context["READTHEDOCS"] = os.environ.get("READTHEDOCS", "") == "True"
```
perhaps with brackets for clarity but I think the syntax is valid without.

The behaviour is slightly different from what's currently written because it will assign `False`, whereas the existing code leaves the `html_context` dictionary without that key.

Reply:
I like this! Would you like to add this fix?

Reply:
https://github.com/MESAHub/mesa/pull/720